libenkf/src/gen_kw_config.c

```diff
@@ -240,6 +240,10 @@ const char * gen_kw_config_get_key(const gen_kw_config_type * config ) {
   return config->key;
 }
 
+const char * gen_kw_config_get_tag_fmt(const gen_kw_config_type * config) {
```

Comment:
We could. I do not really see any reason to expose it, but I wanted to create a Python binding to use in the __repr_-funtion.. I would actually prefer it to be static, but I'm not sure if this is possible with ctypes? I'll check it out